drift/lib/src/runtime/query_builder/schema/view_info.dart

```diff
@@ -0,0 +1,25 @@
+part of '../query_builder.dart';
+
+/// A sqlite view.
+///
+/// In drift, views can only be declared in `.drift` files.
+///
+/// For more information on views, see the [CREATE VIEW][sqlite-docs]
+/// documentation from sqlite, or the [entry on sqlitetutorial.net][sql-tut].
+///
+/// [sqlite-docs]: https://www.sqlite.org/lang_createview.html
+/// [sql-tut]: https://www.sqlitetutorial.net/sqlite-create-view/
+abstract class ViewInfo<Self extends HasResultSet, Row>
+    implements ResultSetImplementation<Self, Row> {
+  ///
+  String get actualViewName;
```

Comment:
IMO we can just drop this and use `entityName`. For the table implementation, `actualTableName` only exists for backwards compatibility, we don't need to add a redundant getter for views too.